development/grafana-setup/entrypoint.sh

```diff
@@ -12,6 +13,9 @@ done
 
 echo "Up!"
 
+echo "Adding database for InfluxDB"
+curl -i -XPOST ${INFLUXDB_API} --data-urlencode "q=CREATE DATABASE sitespeedio"
```

Comment:
The Docker README hasn't been updated to the latest InfluxDB version see https://docs.influxdata.com/influxdb/v1.2/guides/writing_data/#creating-a-database-using-the-http-api that is the correct way. GET is deprecated.